client/my-sites/post-selector/pagination.jsx

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
>Do we need this component separately?

The primary need is to manage the page state within the top-level `<PostSelector />` component, rather than imposing this burden on the developer using the component.

I'd quite like to refactor `<PostSelector />` to accept a `query` prop object instead of individual parameters.